assets/src/blocks/event-attendees/style.css

```diff
@@ -1,11 +0,0 @@
-#editor .event-attendees-ul {
-    list-style-type: none;
-}
-
-.event-attendee-image-wrap-div {
-    display: inline-block;
-}
-
-.event-attendee-avatar-img {
-    margin: 0 1em -.5em 0;
-}
```

Comment:
I moved these into the `EventAttendeesList` component stylesheet and added specificity rules there that ensure the css is the same on the frontend as the editor backend.  Note, this file is still left in place and imported (even though its blank) because the block styles have the component styles as a dependency and this ensures that any styles related to the block are loaded in the correct context.  Right now the same styles are used backend and frontend so there's no splitting of those files yet.